addons/mail/__manifest__.py

```diff
@@ -117,9 +117,11 @@
         'views/mail_blacklist_views.xml',
         'views/mail_menus.xml',
         'views/res_company_views.xml',
+        "data/discuss_shortcode_data.xml",
     ],
     'demo': [
-        'data/discuss_channel_demo.xml',
+        'demo/discuss_channel/discuss_channel_demo.xml',
+        "demo/discuss_shortcode/discuss_shortcode_demo.xml",
```

Comment:
That's not how addons are organized, please refer to https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html

You are adding useless sub directories that make understanding file organization harder.

Until you move discuss code into a real discuss folder, please avoid that kind of useless bordelification of modules.